template/script.sh.erb

```diff
@@ -26,4 +26,4 @@ echo "TIMING - Starting jupyter at: $(date)"
 
 # Launch the Jupyter Notebook Server
 set -x
-jupyter notebook --config="${CONFIG_FILE}" <%= context.extra_jupyter_args %>
+PYTHONPATH=$HOME/.local/lib/python3.6/site-packages jupyter notebook --config="${CONFIG_FILE}" <%= context.extra_jupyter_args %>
```

Comment:
Is the idea with this change that the jupyterlmod extension is installed in your home directory under `PYTHONPATH=$HOME/.local/lib/python3.6/site-packages`?

Reply:
Yes, exactly.

Reply:
We checked and it turned out that those site packages were not being loaded which is surprising because it is part of the default Python load path.